demo/bulk_insert/bulk_insert.py

```diff
@@ -143,6 +169,8 @@ def ProcessNodes(nodes_csv_files):
 				    if depleted_labels > 0:
 					labels.remove_descriptors(depleted_labels)
 					depleted_labels = 0
+				    descriptor.entities_created = reader.line_num - 1
+				    descriptor.print_progress()
 				labels.pending_inserts += 1
```

Comment:
Isn't labels.pending_inserts = sum(desc.pending_inserts for desc in labels.descriptors)?